codesign/bin/codesign.dart

```diff
@@ -28,16 +29,16 @@ const String kGCloudDownloadUploadPathOption = 'google-cloud-download-upload-pat
 /// Otherwise, nothing will be uploaded back for dry run. default value is
 /// true.
 ///
-/// For [kGCloudDownloadUploadPathOption], it is a required parameter to specify the google cloud bucket paths.
-/// This path is a single string concatenated by a '#' sign, the path before the '#' sign should be (1) google cloud bucket prefix to download the remote artifacts,
-/// and the path after the '#' sign should be (2) the cloud bucket prefix to upload codesigned artifact to.
-/// For example, supply '--google-cloud-download-upload-path=ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip#ios-usb-dependencies/libimobiledevice/<commit>/libimobiledevice.zip',
-/// and code sign app will download the artifact at 'ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip', and upload signed artifact back to
-/// 'ios-usb-dependencies/libimobiledevice/<commit>/libimobiledevice.zip'.
+/// For [kGCloudDownloadPathOption] and [kGCloudUploadPathOption], they are required parameter to specify the google cloud bucket paths.
+/// [kGCloudDownloadPathOption] is the google cloud bucket prefix to download the remote artifacts,
+/// [kGCloudUploadPathOption] is the cloud bucket prefix to upload codesigned artifact to.
+/// For example, supply '--google-cloud-download-path=ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip',
+/// and code sign app will download the artifact at 'ios-usb-dependencies/unsigned/libimobiledevice/<commit>/libimobiledevice.zip'
 ///
 /// Usage:
 /// ```shell
-/// dart run bin/codesign.dart [--dryrun] --google-cloud-download-upload-path=flutter/<commit>/android-arm-profile/artifacts.zip#flutter/<commit>/android-arm-profile/artifacts.zip
+/// dart run bin/codesign.dart [--dryrun] --google-cloud-download-path=flutter/<commit>/android-arm-profile/artifacts.zip
```

Comment:
Should this be gs://flutter_infra_release/flutter/<commit>/android-arm-profile/artifacts.zip?

Reply:
Good point, yeah I thought about it and right it's better for user to provide the full path including gs:// . refactored.